src/OrleansCounterControl/OrleansCounterControl.csproj

```diff
@@ -45,8 +45,24 @@
     <WarningsAsErrors>4014</WarningsAsErrors>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Microsoft.CodeAnalysis, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
@jthelin I thought we concluded that there is no reason to fight the tools and manually remove version data from references in .csproj files.